### ISSUE TITLE
Error: List column type "checkbox" could not be found.

### ISSUE THREAD

**Issue by a-salihov** [flo-sch/slideshow-plugin#8]
Hi,

Last version of the plugin which I installed from the plugin market display error:

```
...List column type "checkbox" could not be found.
...
/.../modules/backend/widgets/Lists.php line 900
```
It happens when I click Edit slide or Save Slide form.

**Comment by flo-sch** [comment]
Hey,

Yep, I have to take a look at this,
Unfortunately, I will not be able to work on that before November 16th ;

If you know how to handle it and can create a Pull-Request, I could accept it though.

Otherwise, I'll have a look next week.

Thanks!

**Comment by ujackson** [comment]
Yes having the issue when updating a slideshow.

**Comment by flo-sch** [comment]
Thanks to @Jeime for the fix, I have released a 1.0.8 version which should be available for you right now, with the corrections.

Fixed by #9.
